### PR TITLE
homework-4

### PR DESCRIPTION
**Pull request by okonoval**
hw4

### CODE REVIEW

File: js-core/homeworks/homework-4/src/main.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+}
+displayCategories(categoriesVar);
+/*
+ 1. Напишите функцию которая принимает 2 числа
+ и возвращает массив содержащий числа между первым числом и вторым числом;
+ */
+
+function numbersBetween(a, b) {
+    let myArray = [];
+    let i;
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-4/src/main.js]: it's better to declare variable "i" inside **for** loop statement

File: js-core/homeworks/homework-4/src/main.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+ 2. Перепишите задачу FizzBuzz, но с использованием цикла.
+ Расчет чисел должен идти до 100
+ */
+
+// 1. FizzBuzz 3, 5, 3 && % 5
+
+
+function fizzBuzz(num) {
+    let result;
+    if(num % 3  == 0 && num % 5 == 0){
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-4/src/main.js]: I guess we can reuse solution from class

File: js-core/homeworks/homework-4/src/main.js (hunk 1 above)
Comment (okonoval) [reply js-core/homeworks/homework-4/src/main.js]: fixed

File: js-core/homeworks/homework-4/src/main.js (hunk 2 above)
Comment (okonoval) [reply js-core/homeworks/homework-4/src/main.js]: fixed

**Review by OlegLustenko — APPROVED** [review]

**Pull request merged**